maven-plugins/shade-extensions/src/main/java/io/helidon/shade/transformers/package-info.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2024 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Maven Shade plugin custom transformer for merging Helidon service-registry, config-metadata and service.loader files.
+ */
+package io.helidon.shade.transformers;
```

Comment:
The base package for this extension should be: `io.helidon.build.maven.shade`.

Reply:
Fixed